libs/common/src/billing/services/account/billing-account-profile-state.service.spec.ts

```diff
@@ -38,51 +44,61 @@ describe("BillingAccountProfileStateService", () => {
     return jest.resetAllMocks();
   });
 
-  describe("accountHasPremiumFromAnyOrganization$", () => {
-    it("should emit changes in hasPremiumFromAnyOrganization", async () => {
-      billingAccountProfileState.nextState({
+  describe("hasPremiumFromAnyOrganization$", () => {
+    it("should emit true if they have premium from an organization", async () => {
+      userBillingAccountProfileState.nextState({
         hasPremiumPersonally: false,
         hasPremiumFromAnyOrganization: true,
       });
 
       expect(await firstValueFrom(sut.hasPremiumFromAnyOrganization$)).toBe(true);
     });
 
-    it("should emit once when calling setHasPremium once", async () => {
-      const emissions = trackEmissions(sut.hasPremiumFromAnyOrganization$);
-      const startingEmissionCount = emissions.length;
+    it("should emit false if they do not have premium from an organization", async () => {
+      userBillingAccountProfileState.nextState({
+        hasPremiumPersonally: false,
+        hasPremiumFromAnyOrganization: false,
+      });
+
+      expect(await firstValueFrom(sut.hasPremiumFromAnyOrganization$)).toBe(false);
+    });
 
-      await sut.setHasPremium(true, true);
+    it("should emit false if there is no active user", async () => {
+      await accountService.switchAccount(null);
 
-      const endingEmissionCount = emissions.length;
-      expect(endingEmissionCount - startingEmissionCount).toBe(1);
+      expect(await firstValueFrom(sut.hasPremiumFromAnyOrganization$)).toBe(false);
     });
   });
 
   describe("hasPremiumPersonally$", () => {
-    it("should emit changes in hasPremiumPersonally", async () => {
-      billingAccountProfileState.nextState({
+    it("should return true if the user has premium personally", async () => {
```

Comment:
suggestion: maybe update these to follow our naming convention while you're at it? :)  https://contributing.bitwarden.com/contributing/testing/unit/naming-conventions

```suggestion
    it("returns true when the user has premium personally", async () => {
```